src/openrct2/world/Park.cpp

```diff
@@ -734,15 +732,15 @@ namespace OpenRCT2::Park
 
     void SetForcedRating(int32_t rating)
     {
-        _forcedParkRating = rating;
+        GetGameState().Cheats.forcedParkRating = rating;
```

Comment:
```suggestion
        auto& gameState = GetGameState();
        gameState.Cheats.forcedParkRating = rating;
```
Also adjust the line below.